packages/components/CHANGELOG.md

```diff
@@ -4,12 +4,16 @@
 
 -   `CheckboxControl`, `CustomGradientPicker`, `FormToggle`, : Refactor and correct the focus style for consistency ([#50127](https://github.com/WordPress/gutenberg/pull/50127)).
 
+### Breaking Changes
+
+-   `onDragStart` in `<Draggable>` is now a synchronous function to allow setting additional data for `event.dataTransfer` ([#49673](https://github.com/WordPress/gutenberg/pull/49673)).
```

Comment:
Unfortunately, this is a breaking change, but I think it actually fixes a bug so it might not be that impactful. c.c. @ciampo if you think this is acceptable :).

Reply:
We should probably add a dev note for this change, though. I've added the GH label